UserRelated/ExampleData/createExampData.py

```diff
@@ -0,0 +1,31 @@
+from OsmDataCollector import OsmDataCollector
+import json
+import os
+import shutil
+
+baiersbronn = [8.1584, 48.4688, 8.4797, 48.6291]
+COORS_PATH = 'tracks_gps_points\\'
+
+
+def create_osm_db(area, n=10):
+    """
+    Creates a json with osm tracks data, for testing purpose.
+    :param area: the bounding box of the tracks to be generated.
+    :param n: The number of fake osm recodes to be generated.
+    """
+    if os.path.isdir(COORS_PATH):
+        shutil.rmtree(COORS_PATH)
+    os.makedirs(COORS_PATH)
+
+    osm_data = OsmDataCollector(area)
+    data = {'tracks': []}
+    for i in range(n):
+        if len(osm_data.tracks) > i:
```

Comment:
shouldn't the access to tracks be via a getter?

Reply:
I chose to make the data members of the classes public to avoid the clumsiness of using getters and setters, I think it is reasonable when using python: 
https://stackoverflow.com/questions/1641219/does-python-have-private-variables-in-classes